src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -1117,7 +1118,8 @@ export default {
 	overflow: auto;
 	// Hide container root element while initializing
 	position: absolute;
-	left: -10000px;
+	/* stylelint-disable csstools/use-logical */ /* upstream logic */
+	left: -100vw;
```

Comment:
why is this needed? Cant you just use `inset-inline-start`?

Reply:
`.tribute-container` applies computed styles to position itself with `left`.
With RTL it's going to be `left: <calculated>px; right: -100vw;`, so it will be out of screen